site/content/docs/5.1/components/scrollspy.md

```diff
@@ -127,34 +130,41 @@ Scrollspy also works with nested `.nav`s. If a nested `.nav` is `.active`, its p
       </nav>
     </div>
     <div class="col-8 col-lg-9">
-      <div data-bs-spy="scroll" data-bs-target="#navbar-example3" data-bs-offset="0" class="scrollspy-example-2" tabindex="0">
+      <div data-bs-spy="scroll" data-bs-target="#navbar-example3" data-bs-smooth-scroll="true" class="scrollspy-example-2" tabindex="0">
```

Comment:
We should ass an offset on this example, otherwise doesn't work as expected

Reply:
I'm not sure about the issue here. Need some explanations.